requirements.txt

```diff
@@ -5,4 +5,5 @@ inflection==0.5.1
 pytz
 tabulate==0.8.9
 pytest==7.0.1
-libfwsi-python==20220123
+libfwsi-python==20240315
```

Comment:
This is OK for now, but remember I'm going to move the lib*-python packages to the `full` installation.